docs/source/usage/plugins/phaseSpace.rst

```diff
@@ -37,47 +37,60 @@ Option                                 Usage
 Output
 ^^^^^^
 
+The 2D histograms are stored in ``.hdf5`` files in the ``simOutput/phaseSpace/`` directory.
+A file is created per species, phasespace selection and time step.
+
 Values are given as *charge density* per phase space bin.
-In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume:
+In order to scale to a simpler *charge of particles* per :math:`\mathrm{d}r_i` and :math:`\mathrm{d}p_i` -bin multiply by the cell volume ``dV``.
 
-.. code:: python
+Analysis
+^^^^^^^^
 
-   ps = ... # get data set from h5py
+The easiest way is to load the data in Python:
 
-   # 3D3V example
-   dV = ps.attrs['dV'] * ps.attrs['dr_unit']**3
-   charge_per_bin = ps[:,:] * ps.attrs['sim_unit'] * dV
+.. code:: python
 
-Out-of-Range Behavior
-^^^^^^^^^^^^^^^^^^^^^
+   from picongpu.plugins.phase_space import PhaseSpace
+   import matplotlib.pyplot as plt
+   from matplotlib.colors import LogNorm
+   import numpy as np
 
-Particles that are *not* in the range of ``<ValL>``/``<ValR>`` get automatically mapped to the lowest/highest bin respectively.
-Take care about that when setting your range and during analysis of the results.
 
-Spatial Offset
-^^^^^^^^^^^^^^
+   # load data
+   phase_space = PhaseSpace('/home/axel/runs/foil_001')
+   e_ps, e_ps_meta = phase_space.get('e', ps='ypy', iteration=1000)
 
-The spatial extent of the output might change due to a moving window.
-Additional attributes are provided to retrieve that spatial information:
+   # unit conversion from SI
+   mu = 1.e6  # meters to microns
+   e_mc_r = 1. / (9.1e-31 * 2.9979e8)  # electrons: kg * m / s to beta * gamma
 
-See this python example:
+   # plotting
+   plt.imshow(
+       np.abs(e_ps) * e_ps_meta.dV,
+       extent = e_ps_meta.extent * [mu, mu, e_mc_r, e_mc_r],
+       interpolation = 'nearest',
+       aspect = 'auto',
+       origin='lower',
+       norm = LogNorm()
+   )
 
-.. code:: python
-
-   ps = ... # get data set from h5py
+   # annotations
+   cbar = plt.colorbar()
+   cbar.set_label(r'$Q / \mathrm{d}r \mathrm{d}p$ [$C s kg^{-1} m^{-2}$]')
```

Comment:
REQUIRED/OPTIONAL:
The units are italic here - please use $\mathrm{C s kg ...}$ or something similar

(sorry this was for some reasons not posted yesterday and popped up as not committed today)

Reply:
thx, done